docs/ocis/deployment/ocis_oc10_backend.md

```diff
@@ -184,15 +255,15 @@ clients:
     insecure: yes
     trusted: yes
     redirect_uris:
-      - http://ocis.domain.com/
-      - https://ocis.domain.com/
-      - http://ocis.domain.com/oidc-callback.html
-      - https://ocis.domain.com/oidc-callback.html
-      - http://ocis.domain.com/oidc-silent-redirect.html
-      - https://ocis.domain.com/oidc-silent-redirect.html
+      - http://ocis.example.org/
+      - https://ocis.example.org/
+      - http://ocis.example.org/oidc-callback.html
```

Comment:
oidc requires https ... even if there are redirects on http ports they should not be listed as redirect_uris. the only exception is the desktop client